extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -321,9 +321,14 @@ export class SubscriptionService {
       } else if (entitlements) { // Not null
         entitlementJson = entitlements;
       }
-      const entitlement = entitlementJson ?
-        Entitlement.parseFromJson(entitlementJson) :
-        Entitlement.empty('local');
+
+      let entitlement;
+      if (entitlementJson) {
+        entitlementJson.raw = authData;
```

Comment:
Hmm. I'd rather we pass it as an argument to `parseFromJson` than as a surrogate field. I don't want anyone try to emulate this with their response.

Reply:
Fixed